webknossos/__generate_client.py

```diff
@@ -336,8 +420,9 @@ def set_response_schema_by_example(
         for path_method in path.values()
         if path_method["operationId"] == operation_id
     ][0]
-    make_properties_required(recorded_response_schema)
+    did_require = make_properties_required(recorded_response_schema)
     request_schema["responses"]["200"]["content"] = recorded_response_schema
+    return did_require
```

Comment:
the return type here is `None`, yet `did_require` is returned? Also I’m uncertain about the name, `did_require` sounds a bit like a boolean? maybe `required_keys` or similar?

Reply:
Ah, this was left out from mypy so far, good catch! I added it to typecheck.sh and resolved the type errors.